package/src/hydra-ai/hydra-ai-backend.ts

```diff
@@ -57,4 +67,12 @@ export default class HydraBackend {
 
     return componentChoice;
```

Comment:
nit: can return from aiService call for consistency with `hydrateComponentWithData` fn